src/client/src/ui/analytics/epics/index.ts

```diff
@@ -1,11 +1,15 @@
 import { combineEpics } from 'redux-observable'
 import { publishPositionUpdateEpic } from './analyticsServiceEpic'
 import { analyticsServiceEpic } from './epics'
+import { InteropServices, PlatformAdapter } from 'rt-components'
 
-const epics = [analyticsServiceEpic]
+export default ({ platform }: { platform: PlatformAdapter }) => {
```

Comment:
Perhaps move this to another file and export default the function here because of the convention we follow in index files.

Reply:
Moved out of index now